packages/orm/src/drivers/PostgresDriver.ts

```diff
@@ -152,7 +157,7 @@ export class PostgresDriver implements Driver {
         });
         const { rows } = await knex.raw(sql, bindings);
         for (let i = 0; i < rows.length; i++) {
-          newRows[i].id = rows[i].id;
+          resetAfterFlushedIds.push({ rows: newRows, index: i, id: rows[i].id });
```

Comment:
I'd like to keep this as-is, i.e. still immediately assign the `.id` and maybe do like `newRows[i].op = "flushed"`, to indicate it's no longer "op = pending", and instead has been moved to flushed-in-the-db.